packages/adapter-components/src/filters/referenced_instance_names.ts

```diff
@@ -75,23 +89,24 @@ const getInstanceNameDependencies = (
       return undefined
     })
     .filter(isDefined)
+  const parentFullName = getFirstParentElemId(instance)?.getFullName() ?? undefined
+  if (extendsParentId && parentFullName) {
```

Comment:
nit: I think this should be 
```suggestion
  if (extendsParentId && parentFullName !== undefined) {
```
unless you also want to omit empty strings (if you do please explain why in a comment + add a test for the scenario)